configuration/enable-offline-mode.adoc

```diff
@@ -44,7 +44,7 @@ You can enable offline mode during the installation of {product-title}.
 include::modules/enable-offline-mode-helm.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
-.Additional resources
+==== Additional resources
```

Comment:
I am not sure if it's a workaround that's unique to OCP, but if the *Additional resources* section applies to the preceding module, assemblies largely use `.Additional resources`. The assembly template itself shows this heading as an H2. Are you sure this is the styling that you want?

Reply:
@kalexand-rh I think those are the guidelines.

This changed is based on the following error message:
```
FAIL: additional resources section for assemblies should be `== Additional resources`. Wrong section name was found in the following files:
configuration/enable-offline-mode.adoc
installing/install-ocp-operator.adoc
integration/integrate-with-ci-systems.adoc
integration/integrate-with-google-cloud-storage.adoc
operating/use-system-health-dashboard.adoc
upgrading/upgrade-helm.adoc
```
Originated from https://github.com/redhataccess/pantheon-cmd 

Which led me to have a look at the guidelines at https://redhat-documentation.github.io/modular-docs/#_additional_resources_4
- Assembly template file and the example assemblies do list additional resources as a section heading.

Reply:
@gaurav-nelson The [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-assemblies) say:

> Only use . formatting (.Additional resources) to follow a module in an assembly. Because you cannot use the xrefs in modules, this functions as a trailing include at the assembly level, where the . formatting of the include statement indicates that the resource applies specifically to the module and not to the assembly.

That's why I'm asking about the function of the section in the assembly.  OCP is not using the Pantheon CMD tool, but if you are, use good judgement about which rule set to prefer. :)

Reply:
Thanks @kalexand-rh 

I want to keep this as per PV2 guidelines for now. As I'm trying to get my content published to the portal.